### DISCUSSION TITLE
ONT Sequencing

### DISCUSSION THREAD

**Discussion by pmenzel** [jodyphelan/TBProfiler#144] [General]
Dear Jody,

we are looking into sequencing MTB cultures with Nanopore, first with the MinION and possibly later with the Flongle.

I saw that in your 2019 paper, that you used the SQK-LWB001 kit from ONT. This low input DNA kit is no longer available, so I wanted to ask if you already tried some of the currently available kits?
Probably many cultures would only yield low amounts of DNA, so that one likely needs a PCR-based kit, such as the Rapid PCR Barcoding Kit (SQK-RPB004) or the PCR Sequencing Kit (SQK-PSK004), which has higher yield in Gbp.

Second, the flongle seems quite useful for sequencing one or maybe two multiplexed samples at lower cost. Did you maybe already give it a try?

thanks and best wishes,
Peter

**Comment by jodyphelan** [comment]
Hi  @pmenzel ,

That sounds interesting. We haven't tried with any of those kits or looked into the flongle unfortunately. I assume the data quality is similar?

Jody

**Comment by peflanag** [comment]
Hi, I just wanted to piggy back on this thread. Does the tbprofiler support ONT reads after base calling? I am hoping to start some work on the Flongle and want to compare with my MiSeq runs.

**Comment by jodyphelan** [comment]
Yep if you change the if you add `--platform nanopore` it will switch to using nanopore optimised tools and parameters.
Let me know how you get along, I'd be interested to see how well nanopore performs vs Illumina.

**Comment by peflanag** [comment]
oh amazing. I haven't started yet but planing to soon to compare ONT with previous Illumina. The plan is that for query DR isolates we get in the hospital I can run then quicker on a Flongle than batching on a MiSeq run! I'll keep you posted as and when I try!

**Comment by peflanag** [comment]
Hey Jody,

Just confirm the command would be;

tb-profiler --nanopore profile -1 $sampleLoc/${sample}*R1.fq.gz -2 $sampleLoc/${sample}*R2.fq.gz  -p $sample --dir $outputLoc


I have some Nanopore reads from last year that I was going to play around with!

**Comment by jodyphelan** [comment]
Hi @peflanag,

Almost, usually we would just one set of reads so it should be somehting like:
`tb-profiler profile --platform nanopore  -1 $sampleLoc/${sample}.fq.gz -p $sample --dir $outputLoc`

Jody

**Comment by peflanag** [comment]
Sorry yes! I just did a copy and paste from one of my scripts and forgot that its one set of reads! I assume there's no need to run Unicycler to assemble and I can directly use the Fast5 files that where basecalled to Fastq?

**Comment by jodyphelan** [comment]
Oh no it should just work direct with the fastq data!

**Comment by peflanag** [comment]
Cool, I might give it a go today!

**Reply by jodyphelan** [reply]
Hi @peflanag,

Any luck with the nanopore profiling?

Jody

**Reply by peflanag** [reply]
Hey Jody,

yet to start! Had to write a proposal for the hospital to start it and validate it. Hopefully start it soon. All the paper works been sent off. However I dug out some reads from ONT sequencing the lab did before I joined them so I’m going to run them through it next week when I’m back! 

Sorry for the delay in getting back to you about it!

**Reply by jodyphelan** [reply]
No worries, sounds good!
